modules/tracking/test/test_trackers.cpp

```diff
@@ -567,6 +567,38 @@ TEST_P(DistanceAndOverlap, Scaled_Data_CSRT)
   test.run();
 }
 
+TEST(GOTURN, memory_usage)
+{
+  cv::Rect2d roi(145, 70, 85, 85);
+  cv::Mat frame;
+
+  std::string model = cvtest::findDataFile("dnn/gsoc2016-goturn/goturn.prototxt");
+  std::string weights = cvtest::findDataFile("dnn/gsoc2016-goturn/goturn.caffemodel", false);
+
+  std::string dataPath =
+    "%YAML:1.0\n"
+    "---\n"
+    "prototxt: " + model + "\n"
+    "caffemodel: " + weights + "\n";
```

Comment:
Avoid that. We don't testing parameters serialization here.

Use existed straightforward approach:
https://github.com/opencv/opencv_contrib/blob/e502f3053dbccc43b57cf049250730e39c30307f/modules/tracking/test/test_trackerParametersIO.cpp#L11-L19

Reply:
But I implemented usage of the paths via cv::FileNode. So, this is gonna be no used for the test?

Reply:
Test must be simple and straightforward.

> this is gonna be no used for the test

At least not in this test with the name "memory_usage".

Reply:
Please, check last changes:
I was forced to remove cv::FileNode, but instead used ```  cv::TrackerGOTURN::Params params;```, as @alalek said.
Win32/win64 builds are passed, so I think this test is ready.